### PR TITLE
feat(guides): cleanup the macOS code signing guide

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: docs/guides/bundler/sign-macos.md [hunk 1]
```diff
@@ -96,9 +118,11 @@ jobs:
         APPLE_ID: ${{ secrets.APPLE_ID }}
         APPLE_PASSWORD: ${{ secrets.APPLE_PASSWORD }}
       with:
-        tagName: app-v__VERSION__ # the action automatically replaces \_\_VERSION\_\_ with the app version
+        tagName: app-v__VERSION__
```
Comment (kasper9n) [inline docs/guides/bundler/sign-macos.md]: I'd prefer to keep this comment here

**Review by FabianLars — APPROVED** [review]
The old one was easier on the eyes, but this one provides way more handholding (that's a good thing :P )

**Comment by lucasfernog** [comment]
> The old one was easier on the eyes, but this one provides way more handholding (that's a good thing :P )

Yeah I noticed that too. Maybe I can add a TLDR section just explaining the meaning of the env vars? @FabianLars

**Comment by FabianLars** [comment]
That could work yeah. We just have to be careful that it's not _too much_/bloated or generally too distrating

**Comment by lorenzolewis** [comment]
Do we think this one is good to merge?

**Comment by FabianLars** [comment]
@lorenzolewis Yeah LGTM, or at least good enough to get it out of the way for now :P

**Pull request merged**